charts/rasa-x/templates/app-deployment.yaml

```diff
@@ -50,11 +50,13 @@ spec:
           httpGet:
             path: "{{ .Values.app.endpoints.healthCheckUrl }}"
             port: "http"
+          initialDelaySeconds: {{ .Values.app.initialProbeDelay }}
```

Comment:
Actually, it would be good to make a distinction between the `readinessProbe ` and`livenessProbe`, e.g.

```
livenessProbe:
  initialDelaySeconds: {{ .Values.app.livenessProbe.initialDelaySeconds }}

readinessProbe:
  initialDelaySeconds: {{ .Values.app.readinessProbe.initialDelaySeconds }}
```

The same for the rest of the deployments.